_posts/2020-09-10-rustpython-usecase.md

```diff
@@ -0,0 +1,59 @@
+The RustPython Notebook is a **toy** notebook inspired by the now inactive [Iodide project](https://alpha.iodide.io/). It is a testament to how powerful and inspiring the Iodide project was.  
+You can find the RustPython notebook code on [this link](https://github.com/RustPython/RustPython/tree/master/wasm/notebook) under `RustPython/wasm/notebook`. 
+
+Here is how the notebook looks like:
+![Rust Python Notebook](/assets/media/rustpython-notebook.png)
+
+This post is a background discussion on how RustPython can be used for scientific computing in the browser and why that is important. A follow-up post will include a detailed example that showcases how Javascript and Python can work together.
+
+The hope is that this post inspires you to contribute to RustPython.
+
+## Scientific Computing in the browser
+
+This [blog post](https://hacks.mozilla.org/2019/03/iodide-an-experimental-tool-for-scientific-communicatiodide-for-scientific-communication-exploration-on-the-web/) on Mozilla Hacks frames the need for scientific computing in the browser. It focuses on communication. Scientists need to communicate better with policymakers, journalists or interested citizens. One might argue that there is an even greater need for better and more efficient communication within the scientific community. 
+
+Since Mozilla's post in 2019, a lot has happened! (no joke, there was a global pandemic and an atmosphere of mistrust in science).  
+There is a paramount need to "agree on facts" and have verifiable, data-based and model-driven discussions. At a minimum, there is a need to change how discussions are conducted and move them from being centered around "opinion" and "personality" to being focused on "data" and "models". These models need to be demistified in order to rebuild trust in science. One of the many ways to achieve this is by exposing how models work and let anyone interact with them. Scientists need to meet people in the environement used by most: the browser.
+
+## Sharing scientific work should be as easy and worry-free as sharing a google doc
+Coders and the technical community can help. We need to make it extremly simple for scientists to share their work. A scientist should not be expected to run, manage, secure and pay for a server that runs their model. When someone's code or research paper becomes popular, they should not have to pick up the bill for "cloud computing". Besides, no scientist should have to worry about server uptime and security patches. 
+
+Imagine if we asked people to configure a server every time that they wanted to share a google doc. Imagine that every time you needed to open a document on your computer, you will need to install custom fonts with specific versions and then install and configure yet another version of a PDF reader. Opening old documents would most likely require you to rollback your changes. If anything breaks, your document won't work. On top of everything, all of this would require adminitrative access.
```

Comment:
This is an interesting point. I think Google docs and RustPython notebook work same way for the notebook users. Free and 24/7 opening central servers are not very far from browser-hosted platform for users unless network is not available.

But if this article starts with this:
> The hope is that this post inspires you to contribute to RustPython.
It will be big difference for the contributors.

Reply:
I think this section could use some consistency. I do use a google doc in the sub-title, then switch to using a PDF example in the middle of the paragraph to illustrate a point about local configuration...

I meant to illustrate the example of a worry free and easy to share analogy, so I used google doc. Probably a GitHub repo used to host a static site is technically more accurate. I am not sure if it will resonate as much. I could find a way to mention both.

For rolling back changes, i meant to use an imaginary example case that highlights the redundancy or inefficiency of repeated work when you have to configure a version of a reader/python and fonts/libraries. I think I will take out the part about old document or write it more clearly.